.pre-commit-config.yaml

```diff
@@ -43,9 +43,9 @@ repos:
     hooks:
       - id: eslint
         additional_dependencies:
-          - "eslint@8.5.0"
-          - "eslint-config-prettier@9.0.0"
-          - "@typescript-eslint/eslint-plugin@6.7.2"
-          - "@typescript-eslint/parser@6.7.2"
+          - "eslint@8.56.0"
+          - "eslint-config-prettier@9.1.0"
+          - "@typescript-eslint/eslint-plugin@7.0.1"
+          - "@typescript-eslint/parser@7.0.1"
```

Comment:
How annoying that you can't just use what's declared in package.json.

Reply:
Yeah, I was thinking, if it would make more sens to call `npm run lint <CHANGED_FILES_ARG>` as from pre-commit instead of using the ESLint hook in the future.

Reply:
Sounds reasonable.